proto/sf/substreams/v1/substreams.proto

```diff
@@ -33,16 +33,16 @@ message Request {
 
   Modules modules = 6;
   repeated string output_modules = 7;
-  repeated string initial_store_snapshot_for_modules = 8;
+  repeated string debug_initial_store_snapshot_for_modules = 8; // Available only in developer mode
```

Comment:
The comment must be **strictly** on the top of the fields in `.proto` definition, otherwise the comments are not attached properly to the generate code.